content/Phoenix for Rails developers - Bruno Vezoli.md

```diff
@@ -0,0 +1,347 @@
+# Phoenix for Rails developers: a practical example
+
+![Phoenix logo](images/phoenix-header.png)
+
+I started working on Ruby after an exploratory phase of many languages lead me to it. The syntax just captivated me, it was so expressive that it could be read like English and it had so many tools I could do things more productively than ever before. No more string helper functions or awkward loops to manipulate arrays, you could all those things out of the box and just focus on coding your solution.
+
+After a while I found Rails, and, although it was a bit difficult to get the hang of it, it felt right in with Ruby’s philosophy. I felt much more productive than with other tools and it came with batteries included. No hours of configurations or research for a slightly better library which just required you to do all the work, I could once more focus on what I wanted build and let Rails handle the details for me.
+
+Since that time I’ve tried many other web development frameworks in a wide variety of languages but I never felt at home with any of them. Even if many tried to replicate what Rails did there was always missing, whether it was because of the language or some decision where it diverged they always felt like mere copies to me.
+
+### Phoenix
+
+Enter Phoenix, a web framework built on top of Elixir, a functional language with a Ruby-like syntax that runs on top of the Erlang virtual machine. This framework took after many of the features that make Rails what it is: MVC architecture, an opinionated file structure, included tools such as an ORM, a template engine and a routing layer. But it never tried to be another copy, instead it diverged with time and took many decisions that fit better the functional paradigm and the different language, all without sacrificing productivity or developer happiness.
+
+In time, this led to an independent web framework but with that special flavor we all love from Rails. There’s one difference though, since Elixir was built on top of Erlang, which was designed to build massively scalable applications, all apps built with Phoenix inherit this perk. This means that you can code away without having to worry how your app will perform once you reach the big leagues. This is something with which many Rails apps struggle and it’s the main reason why Jose Valim developed Elixir in the first place.
+
+But enough preaching, let’s see some code!
+
+
+### Comparison by example
+
+This post does not intend to be a complete guide on Phoenix, it’s a big framework with lots of things to get into. Instead, I’ll show a side by side comparison of how things are done in both frameworks using the scaffolding they provide to throw some insight on their best practices with a classic blog example.
```

Comment:
I would move this paragraph to the introduction. I suggested in a comment before to add a new paragraph there, this could be part of it :)